Index: app/scripts/services.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/scripts/services.js	(revision )
+++ app/scripts/services.js	(revision )
@@ -25,7 +25,8 @@
         }
 
         function getUserFromToken() {
-            var token = $localStorage.token;
+            //var token = $localStorage.token;
+            var token = localStorage.token;
             var user = {};
             if (typeof token !== 'undefined') {
                 var encoded = token.split('.')[1];
@@ -48,7 +49,8 @@
             },
             logout: function(success) {
                 changeUser({});
-                delete $localStorage.token;
+                //delete $localStorage.token;
+                delete localStorage.token;
                 success();
             }
         };
\ No newline at end of file
